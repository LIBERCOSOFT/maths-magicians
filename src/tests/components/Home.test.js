import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { BrowserRouter } from 'react-router-dom';
import Home from '../../pages/Home';

test('renders home component snapshot', () => {
  const { asFragment } = render(<Home />, { wrapper: BrowserRouter });
  expect(asFragment()).toMatchSnapshot();
});
