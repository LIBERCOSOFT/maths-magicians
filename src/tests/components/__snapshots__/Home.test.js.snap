// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders home component snapshot 1`] = `
<DocumentFragment>
  <div
    class="home__container"
  >
    <h2>
      Welcome to our page!
    </h2>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla sagittis nibh ut neque sollicitudin vestibulum. Maecenas varius leo quis sem efficitur pharetra. Curabitur posuere ex orci, nec ultricies augue aliquet sed. Pellentesque id arcu et quam ultrices congue nec at lectus. Vestibulum ultrices lacus malesuada aliquet tincidunt. Integer a fermentum quam. Etiam commodo vestibulum metus ut imperdiet. Vivamus congue augue a ex tristique auctor. Ut suscipit tellus nibh, ut porta lorem tincidunt sit amet. Integer ornare nibh mauris, ut pharetra urna mollis et. Phasellus commodo finibus ante sit amet malesuada. Suspendisse porttitor lorem libero, sed ullamcorper nisi pulvinar quis. Vestibulum eu arcu eget arcu aliquam porttitor. Nulla ut scelerisque mauris. Nunc quis urna vitae mauris tincidunt fermentum at vitae odio.
    </p>
    <p>
      Praesent vel bibendum augue. Nullam eu viverra augue. Quisque semper eu neque sit amet mollis. Vivamus at nisl congue, eleifend urna ac, posuere lacus. Quisque lobortis dapibus gravida. Curabitur at lectus porttitor, mollis ex non, sodales ipsum. Aenean tristique non mi eget elementum. Nunc ac vulputate eros, in euismod arcu.
    </p>
  </div>
</DocumentFragment>
`;
